README.md

```diff
@@ -109,6 +109,18 @@ try {
 
 The above function will handle the purchase against Apple App Store or Google Play Store and send the in-app purchase receipt for server-side receipt verification to your Chargebee account. Use the Subscription ID returned by the above function to check for Subscription status on Chargebee and confirm the access - granted or denied.
 
+### Invoke Show Manage Subscriptions Settings
+#### For Android
+The `showManageSubscriptionsSettings()` function is designed to invoke the Manage Subscriptions in your app using Chargebee's Flutter SDKs. `Chargebee.showManageSubscriptionsSettings()`, opens the Play Store App subscriptions settings page.
+#### For iOS
+##### Invoke Upgrade or Downgrade Subscription Flow in your App
```

Comment:
I think we don't need this heading because we already have a `### Invoke Show Manage Subscriptions Settings` heading.

Reply:
Done!